cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -18,12 +19,14 @@
     pytest.mark.meta(server_roles="+automate"),
     test_requirements.ssui,
     pytest.mark.long_running,
-    pytest.mark.ignore_stream("upstream", "5.9"),
     pytest.mark.provider([InfraProvider],
                          required_fields=[['provisioning', 'template'],
                                           ['provisioning', 'host'],
                                           ['provisioning', 'datastore']],
                          scope="module"),
+    pytest.mark.provider([CloudProvider],
+                         required_fields=[['provisioning', 'image']],
+                         scope="module"),
```

Comment:
**REQUIRED**: You can't use ``pytest.mark.provider`` twice, only the later one will be used. You need to use smtg like:
```
infra_prov = ProviderFilter(classes=[InfraProvider], required_fields=[...])
cloud_prov = ProviderFilter(classes=[CloudProvider], required_fields=[...])

pytestmark = [
    ...
    pytest.mark.provider(gen_func=providers, filters=[infra_prov, cloud_prov], scope='module'),
```